eos-phone-home

```diff
@@ -171,9 +172,9 @@ class PhoneHome(object):
             uploading_enabled = config['global'].getboolean('uploading_enabled',
                 uploading_enabled)
             environment = config['global'].get('environment', environment)
-        except Exception as e:
-            print("Unable to read %s, assuming fallback values." %
-                (permissions_file, ), e, file=sys.stderr)
+        except:
+            log.info("Unable to read %s, assuming fallback values.",
+                     permissions_file)
```

Comment:
Ugh. `getboolean` does raise an exception if the value in the file is not well-typed! Okay, I concede that the exception is useful in this case :)